src/apps/openfluid-devstudio/MainWindow.cpp

```diff
@@ -337,72 +329,16 @@ MainWindow::~MainWindow()
 // =====================================================================
 
 
-void MainWindow::onJobsClicked()
-{
-  m_WareBuildOptions.IsParallelJobs = !m_WareBuildOptions.IsParallelJobs;
-  updateBuildJobsStatus();
-}
-
-
-// =====================================================================
-// =====================================================================
-
-
-void MainWindow::onJobsScrolled(bool Up)
-{
-  if (m_WareBuildOptions.IsParallelJobs)
-  {
-    if (Up)
-    {
-      m_WareBuildOptions.JobsNumber++;
-    }
-    else if (m_WareBuildOptions.JobsNumber >= 3)
-    {
-      m_WareBuildOptions.JobsNumber--;
-    }
-    m_WareBuildOptions.writeOptionsInWorkspace();
-    updateBuildJobsStatus();
-  }
-}
-
-
-// =====================================================================
-// =====================================================================
-
-
-void MainWindow::toggleModeBuildOption()
-{
-
-  m_WareBuildOptions.IsReleaseMode = !m_WareBuildOptions.IsReleaseMode;
-  m_WareBuildOptions.writeOptionsInWorkspace();
-  updateConfigureModeStatus();
-}
-
-
-// =====================================================================
-// =====================================================================
-
-
-void MainWindow::toggleInstalledBuildOption()
-{
-  m_WareBuildOptions.IsWithInstall = !m_WareBuildOptions.IsWithInstall;
-  m_WareBuildOptions.writeOptionsInWorkspace();
-  updateBuildModeStatus();
-}
-
-
-// =====================================================================
-// =====================================================================
-
-
 void MainWindow::displayBuildOptionsDialog()
 {
-  openfluid::ui::waresdev::WareBuildOptionsDialog SettingsDialog(m_WareBuildOptions, QApplication::activeWindow());
+  openfluid::waresdev::WareBuildOptions WareBuildOptions;
+  WareBuildOptions.setDefaultsFromWorkspaceManager();
```

Comment:
This should be included in the default beahviour of the WareBuildOptions constructor